source/MRMesh/MRObject.h

```diff
@@ -301,6 +316,8 @@ class MRMESH_CLASS Object : public ObjectChildrenHolder
 
     /// Reads model from file
     MRMESH_API virtual Expected<void> deserializeModel_( const std::filesystem::path& path, ProgressCallback progressCb = {} );
+    /// reads model from oter object
+    MRMESH_API virtual Expected<void> setModelFromObject_( const Object& other );
```

Comment:
lets rename it to something like `setSharedModel_`,  and add comment that it differs form simple `setModel` because some Object types consider different fields as model